python/dgllife/utils/featurizers.py

```diff
@@ -2081,3 +2155,171 @@ def __init__(self, bond_data_field='e', self_loop=False):
                                                              Chem.rdchem.BondStereo.STEREOZ,
                                                              Chem.rdchem.BondStereo.STEREOE])]
             )}, self_loop=self_loop)
+
+class PAGTN_EdgeFeaturizer(object):
+    """The bond featurizer used in PAGTN
+
+    PAGTN is introduced in
+    `Path-Augmented Graph Transformer Network. <https://arxiv.org/abs/1905.12712>`
+
+    We build a complete graph and The Edge features include:
+    * **Shortest path between two nodes in terms of bonds**. Each bond is a One hot
+        encoding of bond type, conjugacy, ring membership
+    * **One hot encoding of family of rings**.
+    * **One hot encoding of the distance between the nodes**.
+
+    **We assume the resulting DGLGraph will be created with :func:`smiles_to_complete_graph` with
+    self loops.**
+
+    Parameters
+    ----------
+    bond_data_field : str
+        Name for storing bond features in DGLGraphs, default to ``'e'``.
+    max_length : int
+        Maximum distance upto to which shortest paths must be considered.
+        Paths shorter than max_length will be padded and longer will be
+        truncated, default to ``5``.
+
+    Examples
+    --------
+
+    >>> from dgllife.utils import PAGTN_EdgeFeaturizer
+    >>> from rdkit import Chem
+    >>> mol = Chem.MolFromSmiles('CCO')
+    >>> bond_featurizer = PAGTN_EdgeFeaturizer(max_length= 1)
+    >>> bond_featurizer(mol)
+    {'e': tensor([[0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
+                  [1., 0., 0., 0., 0., 0., 0., 0., 1., 0., 0., 0., 0., 0.],
+                  [1., 0., 0., 0., 0., 0., 0., 0., 1., 0., 0., 0., 0., 0.],
+                  [1., 0., 0., 0., 0., 0., 0., 0., 1., 0., 0., 0., 0., 0.],
+                  [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
+                  [1., 0., 0., 0., 0., 0., 0., 0., 1., 0., 0., 0., 0., 0.],
+                  [1., 0., 0., 0., 0., 0., 0., 0., 1., 0., 0., 0., 0., 0.],
+                  [1., 0., 0., 0., 0., 0., 0., 0., 1., 0., 0., 0., 0., 0.],
+                  [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.]])}
+    >>> # Get feature size
+    >>> bond_featurizer.feat_size()
+    14
+
+    See Also
+    --------
+    BaseBondFeaturizer
+    CanonicalBondFeaturizer
+    WeaveEdgeFeaturizer
+    PretrainBondFeaturizer
```

Comment:
Missing `AttentiveFPBondFeaturizer`. Also can you add `PAGTNEdgeFeaturizer` in `See Also` of other bond/edge featurizers?